index.html

```diff
@@ -97,6 +97,23 @@
       .usa-stripe:nth-child(odd){
         background-color: #FFFFFF;
       }
+      #japan{
+        display: grid;
+        background-color: #FFFFFF;
+        grid-template-columns: 1fr 1fr 1fr;
+        grid-template-rows: 1fr 1fr 1fr;
+        height: 67vw;  
+        place-items: center;   
+      }
+      .red-japan{
+        background-color: #bc002d;
+        border-radius: 50%;
+        display: grid;
+        grid-column: 2;
+        grid-row: 2;
```

Comment:
Not sure if you need these either -- Let me know if it works, I will check myself tomorrow as well.

Reply:
wow. Yes you are correct. I check it out in the chrome: inspect and it looks just fine.
Will create another pull soon.